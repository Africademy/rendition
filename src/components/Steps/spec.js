/* globals expect, describe, it */
import React from 'react'
import renderer from 'react-test-renderer'
import { Provider, Steps, Step } from '../../../dist'

describe('Steps component', () => {
  it('should match the stored snapshot', () => {
    const component = renderer.create(
      <Provider>
        <Steps m={3} titleText={'Beginners Guide'} onClose={() => null}>
          <Step status='pending' onClick={() => null}>
            Do this
          </Step>
          <Step status='completed' onClick={() => null}>
            And then this
          </Step>
          <Step status='pending' onClick={() => null}>
            And finally this
          </Step>
        </Steps>
      </Provider>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should match the stored snapshot (ordered)', () => {
    const component = renderer.create(
      <Provider>
        <Steps m={3} ordered bordered={false} activeStepIndex={1}>
          <Step status='completed' onClick={() => null}>
            Do this
          </Step>
          <Step status='completed' onClick={() => null}>
            And then this
          </Step>
          <Step status='pending' onClick={() => null}>
            And finally this
          </Step>
        </Steps>
      </Provider>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('Error thrown if ordered is true and activeStepIndex is not set', () => {
    expect(() => {
      renderer.create(
        <Provider>
          <Steps m={3} ordered bordered={false}>
            <Step status='completed' onClick={() => null}>
              Do this
            </Step>
            <Step status='completed' onClick={() => null}>
              And then this
            </Step>
            <Step status='pending' onClick={() => null}>
              And finally this
            </Step>
          </Steps>
        </Provider>
      )
    }).toThrowError('You must specify the activeStepIndex for ordered Steps')
  })

  it('Error thrown if activeStepIndex is too high', () => {
    expect(() => {
      renderer.create(
        <Provider>
          <Steps m={3} ordered bordered={false} activeStepIndex={3}>
            <Step status='completed' onClick={() => null}>
              Do this
            </Step>
            <Step status='completed' onClick={() => null}>
              And then this
            </Step>
            <Step status='pending' onClick={() => null}>
              And finally this
            </Step>
          </Steps>
        </Provider>
      )
    }).toThrowError('activeStepIndex is out of range')
  })
})
