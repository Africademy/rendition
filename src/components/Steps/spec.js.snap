// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Steps component should match the stored snapshot (ordered) 1`] = `
.c0 {
  font-family: 'Source Sans Pro',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c2 {
  box-sizing: border-box;
}

.c3 {
  padding: 16px;
}

.c6 {
  margin-left: auto;
  margin-right: auto;
}

.c9 {
  font-size: 14px;
}

.c11 {
  margin-right: 8px;
  color: #1AC135;
}

.c15 {
  margin-left: 16px;
  margin-right: 16px;
}

.c17 {
  margin-right: 8px;
  color: #00AEEF;
}

.c20 {
  margin-right: 8px;
  color: #bbc8d6;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c16 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  position: relative;
  margin: 0;
  padding: 16px;
  padding-right: 16px;
  box-shadow: 0 0 10px 0 rgba(221,225,240,0.15);
  border-radius: 10px;
  border: 1px solid transparent;
  background-color: white;
  color: inherit;
}

.c19 {
  font-weight: 600;
}

.c1 {
  font-family: 'Source Sans Pro',Arial,sans-serif;
  font-size: 14px;
  color: #2A506F;
}

.c13 {
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  opacity: 1;
  display: inline-block;
}

.c13:active,
.c13:hover {
  color: hsl(196.29999999999995,100%,37.5%);
}

.c14 {
  color: #00AEEF;
}

.c12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 24px;
}

.c18 {
  font-size: 16px;
  height: 24px;
  width: 24px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-variant-numeric: tabular-nums;
}

.c21 {
  font-size: 16px;
  height: 24px;
  width: 24px;
  border: 1px solid;
  border-radius: 50%;
}

.c5 {
  margin: 16px;
}

<div
  className="c0 c1"
>
  <div
    className="c2 c3 c4 c5"
  >
    <div
      className="c2 c6 c7"
    >
      <div
        className="c2 c8"
      >
        <div
          className="c2 c9 c10"
        >
          <div
            className="c2 c11 c12"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-check-circle fa-w-16 "
              data-icon="check-circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <a
            className="c13 c14"
            color="primary.main"
            onClick={[Function]}
          >
            Do this
          </a>
        </div>
        <div
          className="c2 c15 c16"
        >
          <svg
            height="9"
            viewBox="0 0 29.4 9"
            width="29.4"
            x="0"
            xmlns="http://www.w3.org/2000/svg"
            y="0"
          >
            <g
              transform="translate(1.000000, 4.000000)"
            >
              <path
                d="M20-4c0.1,0,0.2,0,0.2,0.1l0,0l7.9,4c0.3,0.1,0.4,0.5,0.2,0.7c-0.1,0.1-0.1,0.2-0.2,0.2l0,0
			l-8,3.9c-0.3,0.1-0.6,0-0.7-0.2c0-0.1-0.1-0.2-0.1-0.2l0,0l0-3.5l-3.9,0C15.2,1,15,0.8,15,0.5l0-0.1C15,0.2,15.2,0,15.5,0l0,0
			l3.9,0l0-3.5C19.4-3.7,19.6-4,20-4L20-4z M7.5-0.1l4,0c0.3,0,0.5,0.2,0.5,0.5s-0.2,0.5-0.5,0.5l0,0l-4,0C7.2,0.9,7,0.7,7,0.4
			C7,0.1,7.2-0.1,7.5-0.1L7.5-0.1z M-0.5-0.2l4,0C3.8-0.1,4,0.1,4,0.4S3.8,0.9,3.5,0.9l0,0l-4,0C-0.8,0.8-1,0.6-1,0.3
			C-1,0-0.8-0.2-0.5-0.2L-0.5-0.2z"
                fill="#5b82a7"
                fillRule="nonzero"
              />
            </g>
          </svg>
        </div>
      </div>
      <div
        className="c2 c8"
      >
        <div
          className="c2 c9 c10"
        >
          <div
            className="c2 c17 c12 fa-layers"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-circle fa-w-16 "
              data-icon="circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <div
              className="c18 fa-layers-text fa-inverse"
            >
              2
            </div>
          </div>
          <div
            className="c19 "
          >
            And then this
          </div>
        </div>
        <div
          className="c2 c15 c16"
        >
          <svg
            height="9"
            viewBox="0 0 29.4 9"
            width="29.4"
            x="0"
            xmlns="http://www.w3.org/2000/svg"
            y="0"
          >
            <g
              transform="translate(1.000000, 4.000000)"
            >
              <path
                d="M20-4c0.1,0,0.2,0,0.2,0.1l0,0l7.9,4c0.3,0.1,0.4,0.5,0.2,0.7c-0.1,0.1-0.1,0.2-0.2,0.2l0,0
			l-8,3.9c-0.3,0.1-0.6,0-0.7-0.2c0-0.1-0.1-0.2-0.1-0.2l0,0l0-3.5l-3.9,0C15.2,1,15,0.8,15,0.5l0-0.1C15,0.2,15.2,0,15.5,0l0,0
			l3.9,0l0-3.5C19.4-3.7,19.6-4,20-4L20-4z M7.5-0.1l4,0c0.3,0,0.5,0.2,0.5,0.5s-0.2,0.5-0.5,0.5l0,0l-4,0C7.2,0.9,7,0.7,7,0.4
			C7,0.1,7.2-0.1,7.5-0.1L7.5-0.1z M-0.5-0.2l4,0C3.8-0.1,4,0.1,4,0.4S3.8,0.9,3.5,0.9l0,0l-4,0C-0.8,0.8-1,0.6-1,0.3
			C-1,0-0.8-0.2-0.5-0.2L-0.5-0.2z"
                fill="#5b82a7"
                fillRule="nonzero"
              />
            </g>
          </svg>
        </div>
      </div>
      <div
        className="c2 c8"
      >
        <div
          className="c2 c9 c10"
        >
          <div
            className="c2 c20 c12 fa-layers"
          >
            <div
              className="c2 c21"
            />
            <div
              className="c18 fa-layers-text"
            >
              3
            </div>
          </div>
          <a
            className="c13 c14"
            color="primary.main"
            onClick={[Function]}
          >
            And finally this
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Steps component should match the stored snapshot 1`] = `
.c21 {
  display: inline-block;
  box-sizing: border-box;
  cursor: pointer;
  font: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin: 0;
  background: transparent;
  overflow: visible;
  text-transform: none;
  color: #00AEEF;
  outline: none;
  border: none;
  padding: 0;
  text-align: inherit;
}

.c0 {
  font-family: 'Source Sans Pro',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c22 {
  font-weight: 600;
  font-size: 14px;
  height: 38px;
  white-space: nowrap;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c22 svg {
  color: inherit !important;
  font-size: 0.925em;
}

.c22:disabled {
  cursor: not-allowed;
}

.c22 {
  color: #00AEEF;
  height: auto;
  font-weight: 600;
  border-radius: 0;
}

.c22:hover:enabled,
.c22:focus:enabled,
.c22:active:enabled {
  box-shadow: none;
  background: none;
  border-color: none;
  color: hsl(196.29999999999995,100%,37.5%);
  opacity: initial;
}

.c22:hover:enabled svg,
.c22:focus:enabled svg,
.c22:active:enabled svg {
  color: hsl(196.29999999999995,100%,37.5%) !important;
}

.c2 {
  box-sizing: border-box;
}

.c3 {
  padding: 16px;
}

.c6 {
  margin-left: 8px;
  margin-right: 38px;
  min-height: 24px;
}

.c10 {
  margin-left: auto;
  margin-right: auto;
}

.c13 {
  font-size: 14px;
}

.c15 {
  margin-right: 8px;
  color: #bbc8d6;
}

.c19 {
  margin-left: 16px;
  margin-right: 16px;
}

.c20 {
  margin-right: 8px;
  color: #1AC135;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  position: relative;
  margin: 0;
  padding: 16px;
  padding-right: 38px;
  box-shadow: 0 0 10px 0 rgba(221,225,240,0.15);
  border-radius: 10px;
  border: 1px solid #DDE1f0;
  background-color: white;
  color: inherit;
}

.c23 {
  position: absolute;
  font-size: 14px;
  padding: 8px;
  top: 18px;
  right: 8px;
}

.c23 svg {
  height: 14px;
  width: 14px;
}

.c8 {
  font-weight: 400;
}

.c9 {
  margin: 0;
  font-size: 14px;
  color: #527699;
}

.c1 {
  font-family: 'Source Sans Pro',Arial,sans-serif;
  font-size: 14px;
  color: #2A506F;
}

.c17 {
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  opacity: 1;
  display: inline-block;
}

.c17:active,
.c17:hover {
  color: hsl(196.29999999999995,100%,37.5%);
}

.c18 {
  color: #00AEEF;
}

.c16 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 24px;
}

.c5 {
  margin: 16px;
}

<div
  className="c0 c1"
>
  <div
    className="c2 c3 c4 c5"
  >
    <div
      className="c2 c6 c7"
    >
      <h6
        className="c8 c9"
      >
        Beginners Guide
      </h6>
    </div>
    <div
      className="c2 c10 c11"
    >
      <div
        className="c2 c12"
      >
        <div
          className="c2 c13 c14"
        >
          <div
            className="c2 c15 c16"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-check-circle fa-w-16 "
              data-icon="check-circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <a
            className="c17 c18"
            color="primary.main"
            onClick={[Function]}
          >
            Do this
          </a>
        </div>
        <div
          className="c2 c19 c7"
        >
          <svg
            height="9"
            viewBox="0 0 29.4 9"
            width="29.4"
            x="0"
            xmlns="http://www.w3.org/2000/svg"
            y="0"
          >
            <g
              transform="translate(1.000000, 4.000000)"
            >
              <path
                d="M20-4c0.1,0,0.2,0,0.2,0.1l0,0l7.9,4c0.3,0.1,0.4,0.5,0.2,0.7c-0.1,0.1-0.1,0.2-0.2,0.2l0,0
			l-8,3.9c-0.3,0.1-0.6,0-0.7-0.2c0-0.1-0.1-0.2-0.1-0.2l0,0l0-3.5l-3.9,0C15.2,1,15,0.8,15,0.5l0-0.1C15,0.2,15.2,0,15.5,0l0,0
			l3.9,0l0-3.5C19.4-3.7,19.6-4,20-4L20-4z M7.5-0.1l4,0c0.3,0,0.5,0.2,0.5,0.5s-0.2,0.5-0.5,0.5l0,0l-4,0C7.2,0.9,7,0.7,7,0.4
			C7,0.1,7.2-0.1,7.5-0.1L7.5-0.1z M-0.5-0.2l4,0C3.8-0.1,4,0.1,4,0.4S3.8,0.9,3.5,0.9l0,0l-4,0C-0.8,0.8-1,0.6-1,0.3
			C-1,0-0.8-0.2-0.5-0.2L-0.5-0.2z"
                fill="#5b82a7"
                fillRule="nonzero"
              />
            </g>
          </svg>
        </div>
      </div>
      <div
        className="c2 c12"
      >
        <div
          className="c2 c13 c14"
        >
          <div
            className="c2 c20 c16"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-check-circle fa-w-16 "
              data-icon="check-circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <a
            className="c17 c18"
            color="primary.main"
            onClick={[Function]}
          >
            And then this
          </a>
        </div>
        <div
          className="c2 c19 c7"
        >
          <svg
            height="9"
            viewBox="0 0 29.4 9"
            width="29.4"
            x="0"
            xmlns="http://www.w3.org/2000/svg"
            y="0"
          >
            <g
              transform="translate(1.000000, 4.000000)"
            >
              <path
                d="M20-4c0.1,0,0.2,0,0.2,0.1l0,0l7.9,4c0.3,0.1,0.4,0.5,0.2,0.7c-0.1,0.1-0.1,0.2-0.2,0.2l0,0
			l-8,3.9c-0.3,0.1-0.6,0-0.7-0.2c0-0.1-0.1-0.2-0.1-0.2l0,0l0-3.5l-3.9,0C15.2,1,15,0.8,15,0.5l0-0.1C15,0.2,15.2,0,15.5,0l0,0
			l3.9,0l0-3.5C19.4-3.7,19.6-4,20-4L20-4z M7.5-0.1l4,0c0.3,0,0.5,0.2,0.5,0.5s-0.2,0.5-0.5,0.5l0,0l-4,0C7.2,0.9,7,0.7,7,0.4
			C7,0.1,7.2-0.1,7.5-0.1L7.5-0.1z M-0.5-0.2l4,0C3.8-0.1,4,0.1,4,0.4S3.8,0.9,3.5,0.9l0,0l-4,0C-0.8,0.8-1,0.6-1,0.3
			C-1,0-0.8-0.2-0.5-0.2L-0.5-0.2z"
                fill="#5b82a7"
                fillRule="nonzero"
              />
            </g>
          </svg>
        </div>
      </div>
      <div
        className="c2 c12"
      >
        <div
          className="c2 c13 c14"
        >
          <div
            className="c2 c15 c16"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-check-circle fa-w-16 "
              data-icon="check-circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <a
            className="c17 c18"
            color="primary.main"
            onClick={[Function]}
          >
            And finally this
          </a>
        </div>
      </div>
    </div>
    <button
      aria-label="Dismiss"
      className="c21 sc-AxheI c22 c23"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      title="Dismiss"
      type="button"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-times fa-w-11 "
        data-icon="times"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 352 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </button>
  </div>
</div>
`;
